email_notification/helpers/email_translations.py

```diff
@@ -0,0 +1,320 @@
+        "sv": "Ny bokning {{reservation_number}} har gjorts för {{unit_name}}",
+    },
+    "textStaffReservationMadeCheckDetails": {
+        "fi": "Voit tarkistaa varauksen tiedot osoitteessa",
+        "en": "You can view the booking at",
+        "sv": "Du kan se bokningen på",
+    },
+    "textStaffReservationMade": {
+        "fi": "Varausyksikköön {{reservation_unit}} on tehty uusi hyväksytty varaus",
+        "en": "A new booking has been confirmed for {{reservation_unit}}",
+        "sv": "En ny bokningsförfrågan för {{reservation_unit}} har bekräftats",
+    },
+    #
+    # STAFF_NOTIFICATION_RESERVATION_REQUIRES_HANDLING
+    "subjectStaffReservationRequiresHandling": {
+        "fi": "Uusi tilavaraus {{reservation_number}} odottaa käsittelyä toimipisteessä {{unit_name}}",
+        "en": "New booking {{reservation_number}} requires handling at unit {{unit_name}}",
+        "sv": "Ny bokningsförfrågan {{reservation_number}} för {{reservation_unit}} väntar på at behandlats",
+    },
+    "textStaffReservationRequiresHandling": {
+        "fi": "Varausyksikköön {{reservation_unit}} on tehty uusi käsittelyä vaativa varauspyyntö",
+        "en": "A booking request for {{reservation_unit}} is waiting for processing",
+        "sv": "En ny bokningsförfrågan för {{reservation_unit}} väntar på at behandlats",
+    },
+    "staffReservationRequiresHandlingCheckDetails": {
+        "fi": "Voit tarkistaa ja käsitellä varauksen osoitteessa",
+        "en": "You can view and handle the booking at",
+        "sv": "Du kan se bokningen på",
+    },
+    #
+    # Reservation common translations
+    "reserveeName": {
+        "fi": "Varaajan nimi",
+        "en": "Name",
+        "sv": "Namn",
+    },
+    "reservationNumber": {
+        "fi": "Varausnumero",
+        "en": "Reservation number",
+        "sv": "Bokningsnummer",
+    },
+    "reservationBegins": {
+        "fi": "Alkamisaika",
+        "en": "From",
+        "sv": "Börjar",
+    },
+    "reservationEnds": {
+        "fi": "Päättymisaika",
+        "en": "To",
+        "sv": "Slutar",
+    },
+    "oClock": {
+        "fi": "klo",
+        "en": "at",
+        "sv": "kl.",
+    },
+    "reservationPrice": {
+        "fi": "Hinta",
+        "en": "Price",
+        "sv": "Pris",
+    },
+    "vatIncluded": {
+        "fi": "sis. alv",
+        "en": "incl. VAT",
+        "sv": "inkl. moms",
+    },
+    "manageReservation": {
+        "fi": "Hallitse varaustasi Varaamossa. Voit perua varauksesi ja tarkistaa varauksen tiedot sekä Varaamon sopimus- ja peruutusehdot",
+        "en": "Manage your booking at Varaamo. You can check the details of your booking and Varaamo's terms of contract and cancellation on the",
+        "sv": "Hantera din bokning på Varaamo. Du kan kontrollera uppgifterna om din bokning samt Varaamos avtals- och avbokningsvillkor på sidan",
+    },
+    "ownReservationsPage": {
+        "fi": "Omat Varaukset -sivulla",
+        "en": "'My bookings' page",
```

Comment:
Minor typographical point but inverted commas / single quotation marks instead of plain apostrophe: `‘My bookings’`